src/actions/notificationsActions.js

```diff
@@ -163,8 +165,17 @@ export const startListeningNotificationsAction = () => {
       contacts: { data: contacts },
     } = getState();
     if (SOCKET && SOCKET.socket && SOCKET.socket.readyState === 1) {
-      SOCKET.onMessage(async response => {
-        const data = JSON.parse(response.data.msg);
+      SOCKET.onMessage((response) => {
+        let data;
+        try {
+          data = JSON.parse(response.data.msg);
+        } catch (e) {
+          // this shouldn't happen, but was reported to Sentry as issue, let's report with more details
+          Sentry.captureMessage('Platform WebSocket notification parse failed', { extra: { response } });
```

Comment:
We could include `e` object into the `extra` param as well

Reply:
and maybe `msg: get(response, 'data.msg')` as Sentry could hit the recursion limit when displaying the raw `response` object

Reply:
👍